eos-license-service.service.in

```diff
@@ -4,3 +4,7 @@ Wants=network.target
 
 [Service]
 ExecStart=/usr/bin/gjs -I @pkgdatadir@ @pkgdatadir@/licenseApp.js
+User=nobody
+ProtectSystem=strict
+ProtectHome=true
+InaccessiblePaths=/sysroot/home
```

Comment:
You should be able to sandbox harder than this. My current sandboxing copypasta is [in Mogwai](https://github.com/endlessm/mogwai/blob/master/mogwai-scheduled/mogwai-scheduled.service.in#L13).

The `InaccessiblePaths=/sysroot/home` is a good one.

Reply:
Thanks for the pointer. Shame to have to copy this everywhere, but we work with what we have. `MemoryDenyWriteExecute=true` did not fly because JIT; all others seem good.

Reply:
Yeah, it does seem to be a bit of a pain, but I’m not sure what systemd could do better here, given that they have to maintain backwards compatibility with old `.service` files (so can’t introduce new sandboxing things as defaults), and given that every service is different, something like `ApplyDefaultSandboxing=yes-please` probably wouldn’t be flexible enough.

I’ll add `InaccessiblePaths=` to my various `.service` files presently.